### PR TITLE
[bitnami/nginx] Add PSP, Role, RoleBinding and ServiceAccount to nginx chart

### PR DESCRIPTION
**Pull request by pavel-kazhavets**
**Description of the change**

Allows enabling creation of pod security policy, role and rolebinding.

**Benefits**

Allows deploying this chart in clusters with PodSecurityPolicy admission controller enabled.

**Possible drawbacks**

**Applicable issues**

  - fixes #

**Additional information**

**Checklist** 
- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)
- [X] If the chart contains a `values-production.yaml` apart from `values.yaml`, ensure that you implement the changes in both files

:warning: Keep in mind that if you want to make changes to the kubeapps chart, please implement them in the [kubeapps repository](https://github.com/kubeapps/kubeapps/tree/master/chart/kubeapps). This is only a synchronized mirror.

### CODE REVIEW

File: bitnami/nginx/templates/_helpers.tpl [hunk 1]
```diff
@@ -112,3 +113,25 @@ nginx: missing-extra-volume-mounts
...
+{{/*
+Return the appropriate apiVersion for podsecuritypolicy.
+*/}}
+{{- define "podsecuritypolicy.apiVersion" -}}
+{{- if semverCompare "<1.10-0" .Capabilities.KubeVersion.GitVersion -}}
+{{- print "extensions/v1beta1" -}}
+{{- else -}}
+{{- print "policy/v1beta1" -}}
+{{- end -}}
+{{- end -}}
```
Comment (dani8art) [inline bitnami/nginx/templates/_helpers.tpl:137]: Could we add this to the `common` chart?

https://github.com/bitnami/charts/blob/master/bitnami/common/templates/_capabilities.tpl

**Review by dani8art — CHANGES_REQUESTED** [review]
Hi @pavel-kazhavets thanks for this PR, please take a look at my comments

Could you add a line break at the end of the files, please?

File: bitnami/nginx/templates/_helpers.tpl (hunk 1 above)
Comment (pavel-kazhavets) [reply bitnami/nginx/templates/_helpers.tpl]: @dani8art done

**Comment by pavel-kazhavets** [comment]
@dani8art Hey, thank you for review! Can you please take a look again?

**Comment by rafariossaa** [comment]
Hi, 
  Could you take a look to the test that are failing ?

**Comment by pavel-kazhavets** [comment]
> Hi,
> Could you take a look to the test that are failing ?

It's failing because I'm using common chart with bumped version which is bumped in the same PR. Is it OK to merge as-is or do I need to split it into two PRs?

**Comment by andresbono** [comment]
> It's failing because I'm using common chart with bumped version which is bumped in the same PR. Is it OK to merge as-is or do I need to split it into two PRs?

It would be better if you split both changes in different PRs. Sorry for the inconvenience.

Once the `bitnami/common` PR is merged into master, this one can be rebased to resolve possible conflicts.

**Comment by dani8art** [comment]
Hi @pavel-kazhavets did you have time to work on this?

**Pull request closed without merging**